File: ChimpokomonPython/chimpokomon/Carnotron.py
from ChimpokomonPython.chimpokomon.Chimpokomon import Chimpokomon


class Carnotron(Chimpokomon):

    def __init__(self, naturaleza, ataque):
        super().__init__(naturaleza, ataque)
        self.nombre = "Carnotron"
        self.nivelVida = 20

File: videojuegoDeChinpokomon/src/chimpokomon/Carnotron.java
package Chimpokomon;
import Ataque.Ataque;
import Naturaleza.Naturaleza;

import java.util.List;

public class Carnotron extends Chimpokomon{

    public Carnotron(List<Ataque> ataques, Naturaleza naturaleza) {
        super(ataques, naturaleza);
        this.setNombre("Carnotron");
        this.setNivelDeVida(20);
    }

    


}
